src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1020,7 +1020,7 @@ private void CheckRestrictedTypeReceiver(BoundExpression expression, CSharpCompi
 
             var receiver = ReplaceTypeOrValueReceiver(methodGroup.Receiver, !method.RequiresInstanceReceiver && !invokedAsExtensionMethod, diagnostics);
 
-            this.CoerceArguments(methodResult, analyzedArguments.Arguments, diagnostics, receiver?.Type, receiver?.GetRefKind());
+            this.CoerceArguments(methodResult, analyzedArguments.Arguments, diagnostics, receiver?.Type, receiver?.GetRefKind(), method.RequiresInstanceReceiver ? GetValEscape(receiver, LocalScopeDepth) : Binder.ExternalScope);
```

Comment:
If `receiver?.GetRefKind()` is `ref`, should we use `GetRefEscape` instead of `GetValEscape`?